src/zck_read_header.c

```diff
@@ -180,6 +179,13 @@ int main (int argc, char *argv[]) {
             if (!digest_uncompressed)
                 digest_uncompressed = "";
 
+	    if (chk == zck_get_first_chunk(zck)) {
+		    printf("STRLEN > = %d\n", strlen(digest_uncompressed) > 0);
```

Comment:
Is this line needed?

Reply:
No, sorry, a debug output I left. Drop it.